applications/GeoMechanicsApplication/custom_constitutive/small_strain_umat_law.cpp

```diff
@@ -620,7 +666,7 @@ Vector& SmallStrainUMAT3DLaw::CalculateValue(ConstitutiveLaw::Parameters& rParam
         rFlags.Set(ConstitutiveLaw::COMPUTE_STRESS, true);
 
         // We compute the stress
-        SmallStrainUMAT3DLaw::CalculateMaterialResponseCauchy(rParameterValues);
+        SmallStrainUMATLaw<TVoigtSize>::CalculateMaterialResponseCauchy(rParameterValues);
```

Comment:
Apparently, we have a redundant template parameter here:
```suggestion
        SmallStrainUMATLaw::CalculateMaterialResponseCauchy(rParameterValues);
```

Reply:
Removed